server/StudySharp.API/Controllers/TheoryBlockController.cs

```diff
@@ -0,0 +1,58 @@
+using System.Threading.Tasks;
+using AutoMapper;
+using MediatR;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using StudySharp.API.Requests.TheoryBlocks;
+using StudySharp.API.Responses.TheoryBlocks;
+using StudySharp.ApplicationServices.Commands;
+using StudySharp.ApplicationServices.Queries;
+using StudySharp.Domain.General;
+
+namespace StudySharp.API.Controllers
+{
+    [Authorize]
+    [ApiController]
+    [Route("api/[controller]")]
+
+    public class TheoryBlockController : ControllerBase
```

Comment:
Add Update and Get (all theory blocks for the course by CourseId) methods to the controller

Reply:
GetTheoryBlocksByCourseId is still missing